info/v1/container.go

```diff
@@ -516,6 +516,31 @@ type FsStats struct {
 	WeightedIoTime uint64 `json:"weighted_io_time"`
 }
 
+// NvidiaGpuStats are measured using NVML.
+type NvidiaGpuStats struct {
```

Comment:
Do we want this to be nvidia-specific?  Would it be better to have a GpuStats that we could identify as an nvidia GPU?  I guess it comes down to whether all GPUs generally have the same metrics, or if Nvidia's metrics are different than AMD's GPU metrics, for example

Reply:
I prefixed this with nvidia because I wasn't really able to find what metrics other GPU vendors expose (AMD/Intel etc.) and what I did find seemed very non-standard.

Not sure what the best way here is. I would prefer to not add a generic API without knowing that it's really generic enough to fit all (most) use cases.

Reply:
SGTM